discovery/consul/consul.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const (
+	watchInterval = 15 * time.Second
```

Comment:
OPT: you might have a bit more agressive value (5 seconds for instances), but it is not a big deal.

On my side, in Consul Template I implemented a dynamic value based on the duration of previous fetch:
```
t0 := time.Now()
srvs, meta, err := catalog.Service*
fetchDuration := time.Now() - t0
if fetchDuration < watchInterval {
  time.sleep( watchInterval - fetchDuration )
}
```
So, you end up having at the maximum a fetch every retry_interval

Reply:
Yep I'll try that in the next patch if this isn't good enough